src/plugins/theme/borderRadius.js

```diff
@@ -5,6 +5,6 @@ module.exports = function () {
   return {
     'none': '0',
     DEFAULT: '0.3rem', // form input border-radius
-    'round': '50%',
+    full: '9999px', // TW convention for circular
```

Comment:
Using TW way for round - this actually works better since 50% sometimes create ellipses.